client/src/App.tsx

```diff
@@ -29,9 +30,10 @@ function App(): JSX.Element {
                 <Route exact path="/dashboard">
                   <Dashboard />
                 </Route>
-                <Route path="*">
+                {/* <Route path="*">
                   <Redirect to="/login" />
-                </Route>
+                </Route> */}
```

Comment:
Please be mindful of commenting out important routes, we don't want to push this into production. This feature should work without having to comment out features.

Reply:
Oh thanks, I commented this because I thought it was redirecting me everytime y accessed the page. I uncommented it now.

Reply:
@moffatethan When I try to go to /my-sitters page it redirects to dashboard, for that reason I commented out these lines

Reply:
This is probably related to Material UI link and react router dom link. You cannot manually type the url or else you will be redirected

Reply:
You need to have a react router dom link in the app to navigate without problem. Please uncomment and we can worry about the navigation later

Reply:
Ok, makes sense. I uncommented this route now. Thanks

Reply:
Well, in Dashboard page I added a Link to my-sitters page, and it continues redirecting when I click on it. 
Why it redirects all paths to login?